src/TTrigger.h

```diff
@@ -124,9 +124,9 @@ class TTrigger : public Tree<TTrigger>
     bool match_perl(char*, const QString&, int, int posOffset = 0);
     bool match_wildcard(const QString&, int);
     bool match_exact_match(const QString&, const QString&, int, int posOffset = 0);
```

Comment:
:thinking: It is a little inconsistent that, say, this method has argument names, yet others here, also with *multiple* arguments of the same type (which is where it helps to disambiguate them), do not...